mantidproject/mantid
# Use system python even if anaconda is in the path

Some users at ORNL have been installing python which breaks `mantidpython` in a variety of ways. This starts the python that mantid was built with. This is only a solution for linux.

**To test:**
1. Activate anaconda (may need to install it first)
2. Start `mantidpython` and see that it happily tries to startup with that python. You can see which version is started by running `import sys;print sys.argv[0]". This cannot be passed via the command line as a`-c` option though.
3. Rerun `cmake to` get the templates filled in
4. Start `mantidpython` again to see the warning
5. Deactivate anaconda, start `mantidpython` and don't see the warning

Also try out `mantidpython --classic` (vanilla python) and `mantidpython notebook` to start a jupyter notebook.

_There is no associated issue_

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
